src/main/res/values-te/strings.xml

```diff
@@ -1,23 +1,51 @@
+    <string name="cancel">రద్దు చేయండి</string>
+    <string name="disable">ఆపివేయి</string>
+    <string name="enable">ప్రారంభించబడింది</string>
+    <string name="exit_message">మీరు ఖచ్చితంగా నిష్క్రమించాలనుకుంటున్నారా?</string>
+    <string name="exit_title">నిష్క్రమించు</string>
+    <string name="file_upload_permission">మీ పరికరంలో ఫైళ్ళను చదవడానికి మీకు అనుమతి అవసరం</string>
+    <string name="file_upload_permission_denied">ఫైల్‌ను చదవడానికి మీకు అనుమతి నిరాకరించబడింది. కొనసాగించడానికి ప్రారంభించండి</string>
+    <string name="forgot_password">
       <a href="https://www.amahi.org/forgot">మీ పాస్‌వర్డ్ మర్చిపోయారా?</a>
    </string>
+    <string name="header_settings">సెట్టింగులు</string>
+    <string name="hint_password">పాస్వర్డ్</string>
+    <string name="intro_desc_2">మేము మీ వీడియోలను ప్లే చేయడానికి, మీ ఫోటోలను చూడటానికి, మీ సంగీతాన్ని వినడానికి మరియు మరెన్నో మిమ్మల్ని అనుమతిస్తాము!</string>
+    <string name="intro_desc_6">మీరు ప్రారంభించడానికి సిద్ధంగా ఉన్నారు, అమాహిని ఉపయోగించినందుకు ధన్యవాదాలు.</string>
+    <string name="intro_desc_phone_1">మీ స్వంత ఫైల్‌లను యాక్సెస్ చేయడానికి మిమ్మల్ని అనుమతించే Android అప్లికేషన్. అమాహి హోమ్ సర్వర్.</string>
+    <string name="intro_desc_phone_3">మీ ఫోటో లైబ్రరీని సులభంగా బ్రౌజ్ చేయండి, మీ ఫోన్ నుండి నేరుగా ఫోటోలను అప్‌లోడ్ చేయండి.</string>
+    <string name="intro_desc_phone_4">ఇంట్లో లేదా వీధిలో మీ సంగీత లైబ్రరీని వినండి.</string>
+    <string name="intro_desc_phone_5">ఎప్పుడైనా, ఎక్కడైనా మీ ఫోన్‌లో మీ వీడియో మూవీ లైబ్రరీని చూడండి.</string>
+    <string name="intro_desc_tv_3">మీ టీవీలో మీ చిత్ర లైబ్రరీని వేగంగా మరియు చక్కని రిజల్యూషన్‌లో బ్రౌజ్ చేయండి.</string>
+    <string name="intro_desc_tv_4">మీ టీవీలోని మీ మ్యూజిక్ లైబ్రరీని వినండి, మీకు ఇష్టమైన ట్యూన్‌లతో పార్టీ చేయడానికి లేదా విశ్రాంతి తీసుకోవడానికి అనువైనది.</string>
+    <string name="intro_desc_tv_5">మీ పెద్ద స్క్రీన్‌లో మీ వీడియో మరియు మూవీ లైబ్రరీని పూర్తి రిజల్యూషన్‌లో చూడండి.</string>
+    <string name="intro_phone_1">Android కోసం అమాహికి స్వాగతం</string>
+    <string name="intro_title_2">మీ HDA ని యాక్సెస్ చేయండి</string>
+    <string name="intro_title_3">మీ ఫోటోలను బ్రౌజ్ చేయండి</string>
+    <string name="intro_title_4">మీ సంగీతాన్ని ప్లే చేయండి.</string>
+    <string name="intro_title_5">మీ చలనచిత్రం మరియు వీడియో లైబ్రరీని అమలు చేయండి.</string>
+    <string name="intro_title_6">రెడీ</string>
+    <string name="intro_tv_1">Android TV కోసం అమాహికి స్వాగతం</string>
+    <string name="introducing_cast">తారాగణం పరిచయం</string>
+    <string name="label_next">తదుపరి</string>
+    <string name="label_pause">మూసివేయి</string>
+    <string name="label_play">ఆన్ చేయండి</string>
+    <string name="label_previous">మునుపటి</string>
+    <string name="language_en">ఇంగ్లీష్</string>
+    <string name="language_fr">ఫ్రెంచ్</string>
+    <string name="language_es">స్పానిష్</string>
+    <string name="language_bn">బెంగాలీ</string>
+    <string name="language_de">జర్మన్</string>
    <string name="header_settings">సెట్టింగులు</string>
```

Comment:
`header_settings` is present twice.

Reply:
Fixed the issue. Let's see CI.

Reply:
Thanks I've fixed the other conflicts too.

Reply:
The CI still fails. Try building the app locally.

Reply:
Yes, I've tried it There is some issue with punjabi language. I'm unable to figure it out.

Reply:
As my changes were initially minor shall I create a new PR for this